jQuery/ajax.js

```diff
@@ -1,82 +1,113 @@
-function ajax(citizens) {
-  var characters = getUniqueValuesForField('character', citizens);
-  var hometowns = getUniqueValuesForField('hometown', citizens);
-  var sizes = getUniqueValuesForField('size', citizens);
-
-  populateSelectWithOptions("#character-select", characters);
-  populateSelectWithOptions("#hometown-select", hometowns);
-  populateSelectWithOptions("#size-select", sizes);
-
-  var selectHandler = function(event) {
-    var params = gatherParams();
-    var filteredCitizens = filter(params, event.data.citizens);
-    redrawContent(filteredCitizens);
+window.hh = {};
+
+window.hh.selectors = {
+  menu: '.menu',
+  content: '.content',
+  nameInput: '#name-input',
+  characterSelect: '#character-select',
```

Comment:
because I'm an avid hater of camelcase, I would do some shit like this:

```js
window.hh.selectors = {
  menu: ".menu",
  content: ".content",
  inputs: {
    name: "#name-input"
  },
  selects: {
    character: "#character-select",
    name: "#name-input",
    // etc.
  }
}
```